internal/vendors/vagrant/vagrant.go

```diff
@@ -102,7 +97,7 @@ func (ib *inBand) BMCConnection() api.BMCConnection {
 }
 
 func (c *bmcConnection) BMC() (*api.BMC, error) {
-	return c.IpmiTool.BMC()
+	return &api.BMC{}, nil
```

Comment:
Shouldnt we at least fill this struct with meaningful informations, same for outband.BMC()

Reply:
done